requirements.in

```diff
@@ -6,31 +6,32 @@ celery[sqs]==5.2.6
 Flask-Bcrypt==1.0.1
 flask-marshmallow==0.14.0
 Flask-Migrate==3.1.0
-git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd1a20a21e682262#egg=Flask-SQLAlchemy==2.3.2.dev20190108
+git+https://github.com/pallets-eco/flask-sqlalchemy.git@aa7a61a5357cf6f5dcc135d98c781192457aa6fa#egg=Flask-SQLAlchemy==2.5.1
 Flask==2.1.2
 click-datetime==0.2
 # Should be pinned until a new gunicorn release greater than 20.1.0 comes out. (Due to eventlet v0.33 compatibility issues)
 git+https://github.com/benoitc/gunicorn.git@1299ea9e967a61ae2edebe191082fd169b864c64#egg=gunicorn[eventlet]==20.1.0
 iso8601==1.0.2
 itsdangerous==2.1.2
 jsonschema[format]==4.5.1
-marshmallow-sqlalchemy==0.28.0
+marshmallow-sqlalchemy==0.28.1
 marshmallow==3.15.0
 psycopg2-binary==2.9.3
 PyJWT==2.4.0
-SQLAlchemy==1.4.36
+SQLAlchemy==1.4.40
 cachetools==5.1.0
 beautifulsoup4==4.11.1
 lxml==4.9.1
-Werkzeug==2.0.3  # pyup: <2.1.0 # later versions are not compatible with the version of flask-sqlalchemy we have pinned
+defusedxml==0.7.1
+Werkzeug==2.1.1
 python-dotenv==0.20.0
 
 notifications-python-client==6.3.0
 
 # PaaS
 awscli-cwlogs==1.4.6
 
-notifications-utils @ git+https://github.com/alphagov/notifications-utils.git@56.0.0
+notifications-utils @ git+https://github.com/alphagov/notifications-utils.git@56.0.2
```

Comment:
The Canadian Notify team found they needed to pull this down and maintain it themselves pretty early, we should be prepared for that. I might need to clone it temporarily to debug redis client issues